[terra-button] VoiceOver in Safari is highlighting the Terra Button in a frame bigger than the button size, when the button is included to a modal dialog. (#3920)
cerner/terra-core: packages/terra-button/CHANGELOG.md

## Unreleased

* Fixed

>   * VoiceOver in Safari was highlighting the Terra Button in a frame bigger than the button size, when the button was included to a modal dialog.

Reviewer: ```suggestion
  * Fixed bug where VoiceOver in Safari highlighting Terra Button within a modal dialog with a frame bigger than the button size.
```
